bears/general/IndentationBear.py

```diff
@@ -54,13 +56,40 @@ def run(self,
         lang_settings_dict = LanguageDefinition(
                             language, coalang_dir=coalang_dir)
         annotation_dict = dependency_results[AnnotationBear.name][0].contents
-        indent_types = dict(lang_settings_dict["indent_types"])
+        # sometimes can't convert strings with ':' to dict correctly
+        if ':' in list(lang_settings_dict["indent_types"]):
+            indent_types = dict(lang_settings_dict["indent_types"])
+            indent_types[':'] = ''
+        else:
+            indent_types = dict(lang_settings_dict["indent_types"])
+
+        encapsulators = (dict(lang_settings_dict["encapsulators"]) if
+                         "encapsulators" in lang_settings_dict else {})
+
+        encaps_pos = []
+        for encapsulator in encapsulators:
+            encaps_pos += self.get_specified_block_range(
+                                   file,
```

Comment:
also, I'm just noticing this now because of the syntax highlight in the comment above:

`file` is a really bad name for a variable if it overwrites a builtin... I know it does so in Python2, unsure about Python3, could you please check?

also, if you refactor this, please in an own commit, ok?